src/main/java/com/github/manolo8/darkbot/utils/Base64Utils.java

```diff
@@ -0,0 +1,13 @@
+package com.github.manolo8.darkbot.utils;
+
+import java.util.Base64;
+
+public class Base64Utils{
+
+    public static String base64Decode(String text) throws Exception {
```

Comment:
As mentioned above: The idea wasn't just to base64 encode/decode, but to also provide methods for InputStream to base64 decoded string, basically overload the method with a InputStream as parameter, and do:
`public static String base64Decode(InputStream stream) throws Exception {`
wich converts the stream into text, then calls base64Decode(string)